en/docs/guides/Provisioning/outbound-provisioning-with-google.md

```diff
@@ -0,0 +1,148 @@
+        
+        !!! info 
+            Remember the location of and the name of this downloaded file as
+            it is required later on in this guide.
+
+5.  Get the Client ID of the service account.
+    1.  Click **IAM and admin \> Service accounts**. Choose **Edit** from the action items corresponding to the service account you just created. 
+        ![edit-service-account](../assets/img/guides/choose-edit.png) 
+
+    2.  Click on **Show domain-wide delegation**. Select **Enable G Suite Domain-wide Delegation**, give a product name of your choice, and click **SAVE**.  
+        ![enable-g-suite-domain-wide-delegation](../assets/img/guides/enable-domain-wide-delegation.png)
+
+    3.  <a name="copy"></a>Click **View Client ID** and copy the value for the Client ID.  
+        ![copy-client-id](../assets/img/guides/view-client-id.png)
+
+6.  Manage the API client access:
+    1.  Go to your domain's admin console via https://admin.google.com.
+    2.  Click **Security**.
+
+        ![admin-console-security](../assets/img/guides/admin-console-security.png) 
+
+    3.  Click **Advanced settings \> Manage API client access**.
+    4.  Fill the following values:
+        1.  Paste the [Client ID value you copied previously](../../guides/outbound-provisioning-with-google#copy)
+            as the value for Client Name.
+        2.  Enter
+            `                             https://www.googleapis.com/auth/admin.directory.user,https://www.googleapis.com/auth/admin.directory.orgunit,https://www.googleapis.com/auth/admin.directory.group                           `
+            as the value for scopes.
+        3.  Click **Authorize**.
+
+        ![manage-api-client-access](../assets/img/guides/manage-api-client.png) 
+
+7.  Enable Admin SDK.
+    1.  On the Open the [Google developers
+        console](https://console.developers.google.com/cloud-resource-manager)
+       , click the menu icon, and click **APIs & Services** > **Dashboards**.
+    2.  Click on **Enable APIs AND Services**.  
+    3.  Search for Admin SDK and click **Enable**.
+
+        ![admin-sdk](../assets/img/guides/admin-sdk.png)
+
+## Configuring the WSO2 Identity Server to use email address as the username
+
+Provisioning is the process of coordinating the creation of user
+accounts, e-mail authorizations in the form of rules and roles, and
+other tasks such as provisioning of resources associated with enabling
+new users.
+
+1.  Download the WSO2 Identity Server from
+    [here](http://wso2.com/products/identity-server/).
+2.  <a name= "email"></a>When you log into Google, you normally use an email address. So, to
+    integrate this with the Identity Server, you need to configure WSO2
+    IS to enable users to log in using their email addresses. In order
+    to do that, follow the steps found in [Using Email Address as
+    the
+    Username](insert-link)
+    topic.
+3.  Restart the Identity Server. Since the username and password of the
```

Comment:
Restart WSO2 IS using the `-Dsetup` parameter. This parameter is required because the username and password of the admin user was updated.